enumfields/fields.py

```diff
@@ -89,7 +90,20 @@ def __init__(self, enum, **options):
 
     def contribute_to_class(self, cls, name):
         super(EnumFieldMixin, self).contribute_to_class(cls, name)
+
+        def update_initial_field(instance, created, update_fields, **kwargs):
+            if not created:
```

Comment:
why you have "if not created". The model first value is None

Reply:
I think that when creating, the initial value is set by descriptor, or not?

Reply:
correct, when model is created values are assigned -> descriptor is called which saves the initial values in this case.

Reply:
well, it can work also without the if-statement and then the initialization of original descriptor will be skipped. I dropped the statement.